src/helpers/slugify.js

```diff
@@ -0,0 +1,13 @@
+const slugify = (input) => {
+  let stringVal = input.trim().toLowerCase();
+  const specialCharacters = 'àáäâèéëêìíïîòóöôùúüûñç·/_,:;';
+  const convertSpecial = 'aaaaeeeeiiiioooouuuunc------';
+  [...specialCharacters].forEach((character, index) => {
+    stringVal = stringVal.replace(new RegExp(specialCharacters.charAt(index), 'g'), convertSpecial.charAt(index));
+  });
+  stringVal = stringVal.replace(/[^a-z0-9 -]/g, '').replace(/\s+/g, '-').replace(/-+/g, '-');
```

Comment:
Can the replacement for spaces and dash be just one statement? Something like 
```suggestion
  stringVal = stringVal.replace(/[^a-z0-9 -]/g, '').replace(/[\s-]+/g, '-');
```

Reply:
okay, I will try that